build-system/tasks/extension-helpers.js

```diff
@@ -30,16 +30,16 @@ const {
   compileJs,
   compileJsWithEsbuild,
   mkdirSync,
+  doBuildJs,
 } = require('./helpers');
 const {analyticsVendorConfigs} = require('./analytics-vendor-configs');
 const {compileJison} = require('./compile-jison');
-const {endBuildStep, watchDebounceDelay, doBuildJs} = require('./helpers');
+const {endBuildStep, watchDebounceDelay} = require('./helpers');
 const {green, red, cyan} = require('kleur/colors');
 const {isCiBuild} = require('../common/ci');
 const {jsifyCssAsync} = require('./css/jsify-css');
 const {log} = require('../common/logging');
 const {parse: pathParse} = require('path');
-const {removeFromClosureBabelCache} = require('../compile/pre-closure-babel');
```

Comment:
Glad to see this go away! It's going to help with my ongoing experiment to see if we can use `TransformCache` in `pre-closure-babel.js` as well. (If you're curious, in-progress branch is [here](https://github.com/ampproject/amphtml/compare/main...rsimha:2021-04-14-PreClosureTransformCache).)

Reply:
Awesome!

Reply:
Experiment worked! (#33864)